### PR TITLE
Convert SCC AB document

### LINKED ISSUE #1: Convert some SCC AB documents
(IGNORE) * ~~https://www.scc.ca/en/system/files/publications/ASB_POV_ASB-Program-Overview_v13_2022-07-05.pdf~~
(IGNORE) * ~~https://www.scc.ca/en/system/files/publications/SIRB_RG_Adoptions_v2_2021-03-04.pdf~~

Only do this please:
* https://www.scc.ca/en/system/files/publications/SIRB_RG_SDO-Accreditation_v4_2021-03-04.pdf

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
Fixes https://github.com/metanorma/scc-ab-docs/issues/1

* I have compiled with ISO flavor to check the encoding.
* I'm not sure about the attribute configuration I used. Can you please take a look?

Thanks!

### CODE REVIEW

File: sources/SIRB_RG/sections/04-reqs_and_guidance.adoc [hunk 1]
```diff
@@ -0,0 +1,960 @@
+
+== Requirements & Guidance
+
+.REQUIREMENTS & GUIDANCE
```
Comment (ronaldtse) [inline sources/SIRB_RG/sections/04-reqs_and_guidance.adoc]: @manuelfuenmayor sorry I forgot to specify -- we want to change this table into real clauses as in the following structure:

```
=== Consensus Process

The SDO shall provide a process to achieve consensus in the development of its standards under SCC's accreditation.

NOTE: Consensus does not imply unanimity.

NOTE: Sustained opposition is not intended to provide a  "`veto`" to any one stakeholder.
```

If there is "No guidance", write "NOTE: No guidance."

Thanks!

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: sources/SIRB_RG/sections/04-reqs_and_guidance.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply sources/SIRB_RG/sections/04-reqs_and_guidance.adoc]: @ronaldtse, what about for cases like this:
capture1

Should I do:
```adoc
=== References to Certification or Administrative Requirements

Administrative requirements relating to conformity assessment, marks
of conformity, or other non- technical issues should be presented
separately from technical requirements.

NOTE: No requirement.
```

Comment (manuelfuenmayor) [reply sources/SIRB_RG/sections/04-reqs_and_guidance.adoc]: There is also this:

capture2

It seems like these sub-sections don't have section titles.

Comment (manuelfuenmayor) [reply sources/SIRB_RG/sections/04-reqs_and_guidance.adoc]: I've done:
```adoc
[[sec_9.2.3]]
==== {blank}

No standard submitted...

NOTE: No guidance
```

Comment (ronaldtse) [reply sources/SIRB_RG/sections/04-reqs_and_guidance.adoc]: Agree with this.

Comment (ronaldtse) [reply sources/SIRB_RG/sections/04-reqs_and_guidance.adoc]: > @ronaldtse, what about for cases like this: capture1
> 
> Should I do:
> 
> ```
> === References to Certification or Administrative Requirements
> 
> Administrative requirements relating to conformity assessment, marks
> of conformity, or other non- technical issues should be presented
> separately from technical requirements.
> 
> NOTE: No requirement.
> ```

I think we should do this:

```adoc
=== References to Certification or Administrative Requirements

No requirement.

NOTE: Administrative requirements relating to conformity assessment, marks
of conformity, or other non- technical issues should be presented
separately from technical requirements.
```

Comment (ronaldtse) [reply sources/SIRB_RG/sections/04-reqs_and_guidance.adoc]: I've done the change in 041c6e4.

**Pull request merged**